core/src/banking_stage.rs

```diff
@@ -372,6 +369,21 @@ impl LikeClusterInfo for Arc<ClusterInfo> {
     }
 }
 
+pub struct BankingStage {
+    cxl: CancellationToken,
```

Comment:
can we give a better name here please? Probably something like validator_exit_signal or something?

Reply:
ce775e791c345111c8269f382fe94b85011f8009